ocdata/combined.py

```diff
@@ -105,13 +105,16 @@ def convert_adsorbate_atoms_to_gratoms(self, adsorbate):
 
         Args:
             adsorbate           An `ase.Atoms` object of the adsorbate
+            bond_indices          A list of integers indicating the indices of the
+                                  binding atoms of the adsorbate
 
         Returns:
             adsorbate_gratoms   An graphic atoms object of the adsorbate.
         """
         connectivity = self.get_connectivity(adsorbate)
         adsorbate_gratoms = catkit.Gratoms(adsorbate, edges=connectivity)
-        adsorbate_gratoms.set_tags([2]*len(adsorbate_gratoms))
+        # tag adsorbate atoms: non-binding atoms as 2, the binding atom(s) as 3
```

Comment:
I'm curious about the value 3. Aren't the allowed `tags` 0 1 and 2, for fixed, non-fixed and adsorbate atoms respectively? 
Or here the tag means something else?

Reply:
So I restructured the adsorbate atoms to have tag 2 if they're non-binding atoms, and 3 if they're binding atom(s). 

The reason is that sometimes due to surface structure, CatKit places the adsorbate out of the unit cell of the surface. & We need to filter out structures like that. The original "is_config_reasonable" 152-156 restricts all atoms of the adsorbate to be within the unit cell. The filtering criteria were too strict and filtered out some possible candidates. I needed the adsorbed atoms of the adsorbate to be within the unit cell, and I need to have a way to track which adsorbate atoms are binding (adsorbed) atoms.  **The bond_indices can no longer be used because once you place the adsorbate on the surface, the indices of the atoms change.

Reply:
Update, I just found out some of the models use these tags, so I will prob look at another possible way to track binding atoms.

Reply:
Fixed! Now all reasonable adslabs configuration outputs have tags 0 1 and 2, for fixed, non-fixed and adsorbate atoms, respectively @sidgoyal78